pandas/core/groupby/grouper.py

```diff
@@ -68,9 +68,34 @@ class Grouper:
         If grouper is PeriodIndex and `freq` parameter is passed.
     base : int, default 0
         Only when `freq` parameter is passed.
+        For frequencies that evenly subdivide 1 day, the "origin" of the
+        aggregated intervals. For example, for '5min' frequency, base could
+        range from 0 through 4. Defaults to 0.
+
+        .. deprecated:: 1.1.0
+            The new arguments that you should use are 'offset' or 'origin'.
+
     loffset : str, DateOffset, timedelta object
         Only when `freq` parameter is passed.
 
+        .. deprecated:: 1.1.0
+            loffset is only working for ``.resample(...)`` and not for
+            Grouper (:issue:`28302`).
+            However, loffset is also deprecated for ``.resample(...)``
+            See: :class:`DataFrame.resample`
+
+    origin : pd.Timestamp, default None
+        The timestamp on which to adjust the grouping. It must be timezone
```

Comment:
`It must be timezone aware if the index of the resampled data is.` -> `The timezone of the timestamp must match the timezone of the index`

Reply:
I did the change. However, I am not sure this entirely true since the `origin` just needs to be timezone aware when the index is. But I agree with you since it is less confusing for the reader anyway.